.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: ruby
 rvm:
-  - 2.4.9
```

Comment:
Given this is open sourced, I'd prefer if we were to keep as many supported minor versions of ruby as possible. We never know in what type of setup the gem will be used. All good for patches version. Now, no need to revert here, but maybe consideration for future bumps.

Reply:
I removed this as Ruby 2.4 support ended 9 days ago. Could've kept it around though. 🤔 
https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

Reply:
Ah but technically `2.4.10` is still modern and secure until the next security bug is found. Will re-add.